infra/aliyun/ali_ecs.go

```diff
@@ -471,3 +474,23 @@ func (a *AliProvider) DeleteSecurityGroup() error {
 	response := ecs.CreateDeleteSecurityGroupResponse()
 	return a.RetryEcsRequest(request, response)
 }
+
+func CreateInstanceTag(tags map[string]string) (instanceTags []ecs.RunInstancesTag) {
+	for k, v := range tags {
+		instanceTags = append(instanceTags, ecs.RunInstancesTag{Key: k, Value: v})
+	}
+	return
+}
+
+func GetAKSKFromEnv(config *Config) error {
```

Comment:
I am wondering if we could define this as `func GetAKSKFromEnv() (string, string, string, error)`.

So this function is decoupled from config.